Bottom: 95bc24b3ea305e93adb662a73687ffb4e731917b
Top:    05d42faa1dddb42bddae6496fb537f9d5fb86dc5
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-09-09 12:23:48 +0100

Build with asserts for polygons


---

diff --git a/src/polygon.c b/src/polygon.c
index 10f62ac..78dda51 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -71,6 +71,7 @@ dicer output is used for HIDs which cannot render things with holes
 /* special polygon editing routines
  */
 
+#undef NDEBUG
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
diff --git a/src/polygon1.c b/src/polygon1.c
index 74cb0cf..ce509d2 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -39,6 +39,7 @@
       are marked
 */
 
+#undef NDEBUG
 #include	<assert.h>
 #include	<stdlib.h>
 #include	<stdio.h>
@@ -102,7 +103,7 @@ int vect_inters2 (Vector A, Vector B, Vector C, Vector D, Vector S1,
 #undef DEBUG_JUMP
 #undef DEBUG_GATHER
 #undef DEBUG_ANGLE
-#undef DEBUG
+#define DEBUG
 #ifdef DEBUG
 #define DEBUGP(...) pcb_fprintf(stderr, ## __VA_ARGS__)
 #else
